Issue #4091: Logging for download sample data failure (#4137)
sunpy/sunpy: changelog/4137.featuredownloadsamplelog.rst

> Added logs for failure while downloading sample data.

Reviewer: ```suggestion
Added log messages when the sample data fails to download.
```
This file should also be named `4137.trival.rst`

Author: Done.
